### ISSUE TITLE
Could not load file or assembly 'MudBlazor, Version=7.0.0.4, Culture=neutral, PublicKeyToken=null

### ISSUE THREAD

**Issue by neozhu** [CodeBeamOrg/CodeBeam.MudExtensions#419]
CodeBeam.MudBlazor.Extensions 7.0.0-preview.4
MudBlazor 7.0.0-rc.2
![image](https://github.com/CodeBeamOrg/CodeBeam.MudBlazor.Extensions/assets/1549611/cdb2a777-2ef7-4e7f-bb6f-cf4d822f2559)

**Comment by mckaragoz** [comment]
You can check with the latest rc version of this library, now released.

**Comment by ziomek64** [comment]
> You can check with the latest rc version of this library, now released.

Is it compatible with latest stable mud blazor?

**Comment by mckaragoz** [comment]
> > You can check with the latest rc version of this library, now released.
> 
> Is it compatible with latest stable mud blazor?

Yes, fully compatible. When core library has breaking changes that affects extension library, we immediately release a new one.

**Comment by ziomek64** [comment]
> > > You can check with the latest rc version of this library, now released.
> > 
> > 
> > Is it compatible with latest stable mud blazor?
> 
> Yes, fully compatible. When core library has breaking changes that affects extension library, we immediately release a new one.

Thank you, I'll try and migrate then

**Comment by mckaragoz** [comment]
The issue should be fixed. Closed this issue, please reopen it if you have further info.
